src/tests/ftest/deployment/critical_integration.py

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/python
+"""
+  (C) Copyright 2018-2021 Intel Corporation.
+
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+"""
+
+import time
+
+from general_utils import run_command, DaosTestError
+from ior_test_base import IorTestBase
+from apricot import TestWithServers
+from command_utils_base import CommandFailure
+
+
+class CriticalIntegration(TestWithServers):
+    """Test Class Description: Verify the basic integration of
+                               the server nodes with available
+                               framework and amongst themselves.
+    :avocado: recursive
+    """
+
+    #TO-DO
+    #Provision all daos nodes using provisioning tool provided by HPCM
+
+    def test_passwdlessssh_versioncheck(self):
+        # pylint: disable=protected-access
+        """
+        Test Description: Verify passwordless ssh amongst the server
+                          server nodes available and verify all server
+                          and client nodes have same daos versions.
+        :avocado: tags=all,deployment,full_regression
+        :avocado: tags=hw,large
+        :avocado: tags=criticalintegration,passwdlessssh_versiocheck
```

Comment:
Is this a typo?  passwdlessssh_versiocheck => passwdlessssh_versioncheck   (missing "n")

Reply:
Yeah it is. Thanks.